### PR TITLE
project layout

### PR DESCRIPTION
**Pull request by ani-sha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
-# graphqlcrud-java
\ No newline at end of file
+# graphqlcrud-java
+
+GraphQLCRUD Java automates schema generation and provides CRUD capabilities.
```
Comment (wtrocki) [inline README.md:3]:
```suggestion
GraphQLCRUD Java automates schema generation and integrates with GraphQL CRUD capabilities offered by https://graphqlcrud.org spec
```

File: src/main/resources/META-INF/resources/index.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```
Comment (wtrocki) [inline src/main/resources/META-INF/resources/index.html:1]: We do not need that file

**Comment by wtrocki** [comment]
Please add:

.gitignore 
remove files that are not required: src/main/java/org/.DS_Store  etc

File: src/main/resources/META-INF/resources/index.html (hunk 2 above)
Comment (ani-sha) [reply src/main/resources/META-INF/resources/index.html:1]: yes I have removed it.

**Pull request merged**